### PR TITLE
MIME4J 0.8.3 Release: add the release to the download page

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
> no changelog?

We don't have a changelog for MIME4J - maybe we should set one up in that git repository?

Do you want me to do it?

> no news?

Will do

**Comment by mbaechler** [comment]
A changelog would be awesome

**Comment by chibenwa** [comment]
@mbaechler can you also upvote the release?

**Comment by chibenwa** [comment]
Here is the new changelog: https://github.com/apache/james-mime4j/pull/27

**Comment by chibenwa** [comment]
Capture d’écran de 2019-03-19 10-32-08

File: src/homepage/_posts/2019-03-19-mime4j-0.8.3.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ - MIME4J-280: Improve exception handling
+ - MIME4J-283: DecoderUtil performance fix
+
+The library is available as a maven artifact, for instance for mime4j core:
+
+```
+<dependency>
+    <groupId>org.apache.james</groupId>
+    <artifactId>apache-mime4j-core</artifactId>
+    <version>0.8.2</version>
```
Comment (mbaechler) [inline src/homepage/_posts/2019-03-19-mime4j-0.8.3.markdown]: 0.8.3

**Comment by chibenwa** [comment]
Merged

**Comment by chibenwa** [comment]
Website deployed

**Pull request closed without merging**